backend/app/adapter/sqldb/short_link_integration_test.go

```diff
@@ -277,8 +278,8 @@ func TestShortLinkSql_IsAliasExist(t *testing.T) {
 }
 
 func TestShortLinkSql_GetShortLinkByAlias(t *testing.T) {
-	twoYearsAgo := mustParseTime(t, "2017-05-01T08:02:16-07:00")
-	now := mustParseTime(t, "2019-05-01T08:02:16-07:00")
+	twoYearsAgo := must.Time(t, "2017-05-01T08:02:16-07:00")
```

Comment:
We should create a helper in ptr package which converts `time.Time` to `*time.Time` and embed the parsing directly in the test cases to improve readability.

Reply:
We should do this for all test cases referenced in this PR!